src/ontology/edits/oeo-physical.omn

```diff
@@ -10664,14 +10664,22 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/608",
 Class: OEO_00050016
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Final energy consumption is an energy consumption value accounting for the energy delivered to and consumed by end users."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Final energy consumption value is an energy consumption value expressing the magnitude of the energy delivered to and consumed by end users."@en,
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A final energy consumption value is an energy consumption value expressing the magnitude of the energy delivered to and consumed by end users."@en,
```

Reply:
Please keep in mind, that we aligned the definitions of final energy consumption (value) and primary consumption (value). So if we change the definition of final energy consumption (value) then we should adapt also primary energy consumption (value).

Also we should keep `final energy consumption` at least as alternative term as this is the term that is used in the domain!

Reply:
> Please keep in mind, that we aligned the definitions of final energy consumption (value) and primary consumption (value). So if we change the definition of final energy consumption (value) then we should adapt also primary energy consumption (value).

I agree. 
In this context, we should also fix the axiom of `primary energy consumption (value)`. Since it is a quantity value, I think we should avoid "has part". Instead: `not (is about some non-energy use)`. The other part will be inherited fom energy consumption value.

Reply:
I am fine with the proposal for the axiomatic change of primary energy consumption.